app/Http/Controllers/Api/CityController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\StoreCityRequest;
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class CityController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(Request $request): JsonResponse
+    {
+        // Filter by airline (show cities that have flights originating from or destined for a specific airline)
+
+        //! Error: when validation fails, laravel redirects to '/' (Validating url query params)
+        $validated = $request->validate([
+            'page_size' => 'nullable|integer|min:1|max:50',
+            'sort_by' => 'nullable|in:id,name',
+            'order' => 'nullable|in:asc,desc',
+        ]);
```

Comment:
It is better to add this logic in a form request as you did with StoreCityRequest

Reply:
When you have multiple params you can add a DTO (data transfer object) in the form request here you have an example: https://lightit.slite.com/app/docs/flX6ZukuAlkM95